INFO:     Started server process [5492]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:3006 (Press CTRL+C to quit)
INFO:     127.0.0.1:59330 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59356 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59361 - "OPTIONS /upload HTTP/1.1" 200 OK
INFO:     127.0.0.1:59361 - "POST /upload HTTP/1.1" 200 OK
INFO:     127.0.0.1:59361 - "GET /processed/f7c0372c-e4b1-4230-93fe-c4763849fdbe HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/uvicorn/protocols/http/h11_impl.py", line 428, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/fastapi/applications.py", line 276, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/applications.py", line 122, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/errors.py", line 184, in __call__
    raise exc
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/fastapi/routing.py", line 266, in app
    response = actual_response_class(content, **response_args)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/responses.py", line 196, in __init__
    super().__init__(content, status_code, headers, media_type, background)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/responses.py", line 55, in __init__
    self.body = self.render(content)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/responses.py", line 199, in render
    return json.dumps(
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
ValueError: Out of range float values are not JSON compliant
INFO:     127.0.0.1:59371 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59379 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59379 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59379 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59382 - "GET /templates HTTP/1.1" 200 OK
INFO:     127.0.0.1:59382 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59392 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59392 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59392 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59392 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59405 - "POST /upload HTTP/1.1" 200 OK
INFO:     127.0.0.1:59405 - "GET /processed/5034d88f-5356-4c89-bc0f-749602e608f4 HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/uvicorn/protocols/http/h11_impl.py", line 428, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/fastapi/applications.py", line 276, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/applications.py", line 122, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/errors.py", line 184, in __call__
    raise exc
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/fastapi/routing.py", line 266, in app
    response = actual_response_class(content, **response_args)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/responses.py", line 196, in __init__
    super().__init__(content, status_code, headers, media_type, background)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/responses.py", line 55, in __init__
    self.body = self.render(content)
  File "/Users/mikespacek/Library/Python/3.9/lib/python/site-packages/starlette/responses.py", line 199, in render
    return json.dumps(
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
ValueError: Out of range float values are not JSON compliant
INFO:     127.0.0.1:59440 - "GET /health HTTP/1.1" 200 OK
INFO:     127.0.0.1:59475 - "GET /health HTTP/1.1" 200 OK
